python/triqs/gf/tools.py

```diff
@@ -259,8 +259,94 @@ def fit_legendre(g_t, order=10):
     l = np.arange(len(lmesh))
     scale = np.sqrt(2.*l + 1) / mesh.beta
     scale = scale.reshape([len(lmesh)] + [1]*len(g_t.target_shape))
-    
+
     g_l = Gf(mesh=lmesh, target_shape=g_t.target_shape)
     g_l.data[:] = c_l.reshape(g_l.data.shape) / scale
 
     return g_l
+
+
+def discretize_bath(delta_inp, Nb , bandwidth = 3, t_val = 0.0, tol=1e-5):
+    """
+    discretizes a given Delta_iw with Nb bath sites
```

Comment:
Provide here the analytic form of the discretized hybridization function and the norm used for the optimization.
Mention also 'scipy.optimize.minimize'.

Reply:
did that